src/containers/BooksList.js

```diff
@@ -1,10 +1,12 @@
 /* eslint-disable react/jsx-key */
 /* eslint-disable react/forbid-prop-types */
 import React from 'react';
+import { connect } from 'react-redux';
 import PropTypes from 'prop-types';
 import Book from '../components/Book';
 
-export default function BooksList({ books }) {
+function BooksList(props) {
+  const { state } = props;
```

Comment:
Type checking props are required for this project. Instead of disabling it, kindly learn about this feature and use it. 

Few suggestions:
- I would recommend using `books` as your props as it's more semantic.
- `PropTypes.array` seems to be ambiguous, why not use `PropTypes.arrayOf`? This way you can specify the content type of the array, for example, an array of objects can be `PropTypes.arrayOf(PropTypes.shape({}))`
- This is the same for ` PropTypes.object`, please use ` PropTypes.shape({})` instead.

See https://reactjs.org/docs/typechecking-with-proptypes.html for more details.